src/lib/plugins/entities.js

```diff
@@ -170,6 +171,18 @@ module.exports.entity=function(entity,serv){
     if(entity.position.distanceTo(entity.lastPositionPlayersUpdated)>2*32)
       entity.updateAndSpawn();
   });
+  
+  entity.on("inventoryChange", function(){
```

Comment:
this should be in a player plugins
and use the "windowUpdate" event instead of creating a new "inventoryChange" event (see https://github.com/PrismarineJS/prismarine-windows#window-windowupdate-slot-olditem-newitem )